.circleci/config.yml

```diff
@@ -93,6 +93,13 @@ workflows:
               ignore: /.*/
       - hmpps/helm_lint:
           name: helm_lint
+      - hmpps/database_schema_report:
+          schema: "public"
+          initialise_database: "./gradlew check"
```

Comment:
this will rerun all the tests again, ktlint, detekt etc, which it would be good to avoid.. think this is why they just chose to implement the single test / gradle task